src/main/java/seedu/address/model/consultation/Time.java

```diff
@@ -0,0 +1,74 @@
+package seedu.address.model.consultation;
+
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Objects;
+
+/**
+ * Represents a Time in the system.
+ * Guarantees: immutable; is valid as declared in {@link #isValidTime(String)}
+ */
+public class Time {
+
+    public static final String MESSAGE_CONSTRAINTS = "Times should be in the format HH:MM, "
+            + "where hour is between 00 and 23, and minute between 00 and 59.";
```

Comment:
Consider adding an example of a valid time. User may not understand what "HH:MM" means. This helps us to avoid ambiguity.